contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -1145,7 +1145,7 @@ select cast(t.typname as sys.sysname) as name
     as is_nullable
   -- CREATE TYPE ... FROM is implemented as CREATE DOMAIN in babel
   , CAST(1 as sys.bit) as is_user_defined
-  , CASE tsql_type_name
+  , CASE tsql_base_type_name
```

Comment:
Please make the same changes in upgrade file also.

Reply:
Done